terraform/workspace-variables/review_backend.tfvars

```diff
@@ -0,0 +1,4 @@
+resource_group_name  = "s121d01-register"
+storage_account_name = "s121d01registerterraform"
+container_name       = "tfstate"
+#key                  = "qa-bat.terraform.tfstate"
```

Comment:
Can you replace comment with something along the lines of
```
#key not specified here, must be passed to terraform dymamically
```

Reply:
done ...